### PR TITLE
fix pact bug for circlemargin arcmargin cosmargin

### PR DESCRIPTION
**Pull request by RainFrost1**

### CODE REVIEW

File: ppcls/configs/Vehicle/ResNet50.yaml [hunk 1]
```diff
@@ -51,11 +51,8 @@ Optimizer:
   name: Momentum
   momentum: 0.9
   lr:
-    name: MultiStepDecay
+    name: Cosine
     learning_rate: 0.01
-    milestones: [30, 60, 70, 80, 90, 100, 120, 140]
-    gamma: 0.5
-    verbose: False
     last_epoch: -1
```
Comment (Intsigstephon) [inline ppcls/configs/Vehicle/ResNet50.yaml]: 这个还需要吗？

File: ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  name: "RecModel"
+  infer_output_key: "features"
+  infer_add_softmax: False
+  Backbone: 
+    name: "ResNet50_last_stage_stride1"
+    pretrained: True
+  BackboneStopLayer:
+    name: "adaptive_avg_pool2d_0"
+  Neck:
+    name: "VehicleNeck"
```
Comment (Intsigstephon) [inline ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml:33]: 这儿崔老师提了一个PR，会去掉VehicleNeck.  等他那个合入了，得做下修改

File: ppcls/configs/slim/ResNet50_vehicle_cls_quantization.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  name: "RecModel"
+  infer_output_key: "features"
+  infer_add_softmax: False
+  Backbone: 
+    name: "ResNet50_last_stage_stride1"
+    pretrained: True
+  BackboneStopLayer:
+    name: "adaptive_avg_pool2d_0"
+  Neck:
+    name: "VehicleNeck"
```
Comment (Intsigstephon) [inline ppcls/configs/slim/ResNet50_vehicle_cls_quantization.yaml:32]: 同上

File: ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    class_num: 30671
+    margin: 0.15
+    scale: 32
+ 
+# loss function config for traing/eval process
+Loss:
+  Train:
+    - CELoss:
+        weight: 1.0
+    - SupConLoss:
```
Comment (Intsigstephon) [inline ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml:50]: 这个 loss, 需要和PK sampling配合使用； 胜禹说测出了一点问题，要不再确认下？

File: ppcls/configs/slim/ResNet50_vehicle_cls_quantization.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    class_num: 431
+    margin: 0.15
+    scale: 32
+ 
+# loss function config for traing/eval process
+Loss:
+  Train:
+    - CELoss:
+        weight: 1.0
+    - SupConLoss:
```
Comment (Intsigstephon) [inline ppcls/configs/slim/ResNet50_vehicle_cls_quantization.yaml:47]: 这个loss, 需要和PK Sampling一起使用； 胜禹说实现上有点问题，再确认下

File: ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+              order: ''
+          - RandomErasing:
+              EPSILON: 0.5
+              sl: 0.02
+              sh: 0.4
+              r1: 0.3
+              mean: [0., 0., 0.]
+
+    sampler:
+        name: DistributedRandomIdentitySampler
```
Comment (Intsigstephon) [inline ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml:97]: 名字太长了，建议改下，比如叫DistributedPKSampler

File: ppcls/configs/Vehicle/ResNet50.yaml (hunk 1 above)
Comment (RainFrost1) [reply ppcls/configs/Vehicle/ResNet50.yaml]: 没太理解

File: ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml (hunk 2 above)
Comment (RainFrost1) [reply ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml:33]: 先合入这个吧，那个pr还得再改一版

File: ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml (hunk 4 above)
Comment (RainFrost1) [reply ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml:50]: bug跟配置文件应该没有关系，这个地方应该不需要修改

File: ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml (hunk 6 above)
Comment (RainFrost1) [reply ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml:97]: 这个最好让类的作者或者胜禹修改的pr统一改，每个pr新增的功能尽量统一

File: ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml (hunk 2 above)
Comment (Intsigstephon) [reply ppcls/configs/slim/ResNet50_vehicle_cls_prune.yaml:33]: OK，可以。

File: ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml (hunk 4 above)
Comment (Intsigstephon) [reply ppcls/configs/slim/ResNet50_vehicle_reid_quantization.yaml:50]: OK

**Review by cuicheng01 — APPROVED** [review]
